airflow/models/xcom.py

```diff
@@ -688,16 +684,20 @@ def _deserialize_value(result: XCom, orm: bool) -> Any:
         if orm:
             object_hook = XComDecoder.orm_object_hook
 
-        if result.value is None:
+        value = result.value
+        if isinstance(value, memoryview):
+            value = value.tobytes()
```

Comment:
A side effect of converting the XCom SQLAlchemy code to New Style. SQLAlchemy seems to return `memoryview` instead of `bytes` for binary blob columns (`XCom.value`).

Reply:
Found another way to do this instead.